packages/preview-middleware/src/base/flp.ts

```diff
@@ -271,11 +271,14 @@ export class FlpSandbox {
             : '@sap-ux/preview-middleware';
         const config = { ...this.templateConfig };
         /* sap.ui.rta needs to be added to the list of preload libs for variants management and adaptation projects */
-        if (!config.ui5.libs.includes('sap.ui.rta')) {
-            const libs = config.ui5.libs.split(',');
-            libs.push('sap.ui.rta');
-            config.ui5.libs = libs.join(',');
-        }
+        const preloadUI5Lib = ['sap.ui.rta', 'sap.fe.core', 'sap.fe.templates'];
```

Comment:
we should only do that if the app is a v4 app.

Reply:
I reverted the change. It was the wrong spot to add a lib. Here we are adding additional libs required for the flexibility editor. Getting the preload libs depending on the apps happens in `getUI5Libs` in the same file.